tests/data/strings.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" ?>
+<!-- Test translation file.
+   - Any copyright is dedicated to the Public Domain.
+   - http://creativecommons.org/publicdomain/zero/1.0/ -->
+<!DOCTYPE resources [ <!ENTITY foo "Foo"> <!ENTITY bar "Bar &foo;"> ]>
+<resources>
+  <string name="one"></string>
+  <string name="two"/>
+  <!-- bar -->
+  <string name="three">value</string>
+  <!-- bar -->
+  <!-- foo -->
+  <string name="four">multi-line comment</string>
+  <string name="five" translatable="false">@string/three</string>
+
+  <!-- standalone -->
+
+  <string name="welcome">Welcome to <b>&foo;</b>!</string>
+  <string name="placeholders">Hello, %1$s! You have %2$d new messages.</string>
+  <string name="escape_html">Hello, %1$s! You have &lt;b>%2$d new messages&lt;/b>.</string>
+
+  <string name="ws_trimmed"> &#32; &#8200; &#8195;</string>
+  <string name="ws_quoted">" &#32; &#8200; &#8195;&quot;</string>
+  <string name="ws_escaped"> \u0032 \u8200 \u8195</string>
+
```

Comment:
I would also expect apostrophes to be tested and handled properly:
https://github.com/mozilla/pontoon/pull/3020/files#diff-e8bac283d0aa4db32a8a4a7f254bc66cadeeb611975a0d1beb8d5ecb4979dc80R324